test/e2e/common/volumes.go

```diff
@@ -115,21 +103,13 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 
 	framework.KubeDescribe("NFSv3", func() {
 		It("should be mountable for NFSv3 [Volume]", func() {
-			config := framework.VolumeTestConfig{
-				Namespace:   namespace.Name,
-				Prefix:      "nfs",
-				ServerImage: framework.NfsServerImage,
-				ServerPorts: []int{2049},
-			}
-
+			config, _, serverIP := framework.NewNFSServer(c, namespace.Name, []string{})
```

Comment:
That's true. _common/volumes.go_ and _storage/volumes.go_ are older tests and use defers to cleanup rather than AfterEach. The [`framework.VolumeTestCleanup` ](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/volume_util.go#L233) func deletes the storage server.